stubs/seaborn/seaborn/_core/typing.pyi

```diff
@@ -0,0 +1,24 @@
+from _typeshed import Incomplete
+from collections.abc import Hashable, Iterable, Mapping
+from datetime import date, datetime, timedelta
+from typing import Any
+from typing_extensions import TypeAlias
+
+from matplotlib.colors import Colormap, Normalize
+from numpy import ndarray
+from pandas import DataFrame, Index, Series, Timedelta, Timestamp
+
+ColumnName: TypeAlias = str | bytes | date | datetime | timedelta | bool | complex | Timestamp | Timedelta
+Vector: TypeAlias = Series[Any] | Index[Any] | ndarray[Any, Any]
+VariableSpec: TypeAlias = ColumnName | Vector | None
+VariableSpecList: TypeAlias = list[VariableSpec] | Index[Any] | None
+DataSource: TypeAlias = DataFrame | Mapping[Hashable, Vector] | None
+OrderSpec: TypeAlias = Iterable[str] | None
+NormSpec: TypeAlias = tuple[float | None, float | None] | Normalize | None
+PaletteSpec: TypeAlias = str | list[Incomplete] | dict[Incomplete, Incomplete] | Colormap | None
+DiscreteValueSpec: TypeAlias = dict[Incomplete, Incomplete] | list[Incomplete] | None
+ContinuousValueSpec: TypeAlias = tuple[float, float] | list[float] | dict[Any, float] | None
+
+class Default: ...
```

Comment:
Any reason not to include Deprecated?

Reply:
Looks like there is a new release where this was added :)
I'll update the PR.